unity-renderer/Assets/Desktop/Scripts/MainScripts/DCL/MainDesktop/MainDesktop.cs

```diff
@@ -91,7 +91,7 @@ protected override void InitializeCommunication()
 #endif
 
                 int endPort = startPort + 100;
-                kernelCommunication = new WebSocketCommunication(withSSL, startPort, endPort);
+                kernelCommunication = new WebSocketCommunication(withSSL, 5000, endPort);
```

Comment:
is it ok that you changed `startPort` to `5000`, but `endPort` is still depending on old value?

Reply:
this is just for testing purposes, since im unable to test in. desktop anymore due websocket errors.. will roll back after its fixed